bedrock/firefox/firefox_details.py

```diff
@@ -157,28 +167,25 @@ def get_download_url(self, platform, language, version, product='firefox'):
             language = 'ja-JP-mac'
 
         if version == self.latest_version('aurora'):
-            return self._get_aurora_download_url(platform, language, version)
+            # Download links are different for localized versions
+            if language == 'en-US':
+                if platform == 'Windows':
+                    product = 'firefox-aurora-stub'
+                else:
+                    product = 'firefox-aurora-latest-ssl'
```

Comment:
It's correct as per [download_buttons.py](https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/helpers/download_buttons.py#L65-L72).